docker/step/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN /bin/bash -c 'if [ "$FIND_FASTED_MIRROR" == "true" ]; then source find_faste
   && echo "Choose the fastest PIP source ..." \
   && choose_fastest_pip_source; fi' && \
   apt-get update && \
-  apt-get -qq install -y --no-install-recommends openjdk-8-jre-headless python3 libmysqlclient20 python3-idna libgomp1 python3-setuptools python3-pip && \
+  apt-get -qq install -y --no-install-recommends openjdk-8-jre-headless python3 libmysqlclient20 python3-idna libgomp1 python3-setuptools python3-pip build-essential python3-dev && \
```

Comment:
Do we need both `python3` and `python3-dev`?

Reply:
python3-dev has dependency on python3. Installing python3-dev will trigger the python3 installation automatically. Keeping both is also fine because it won't bring additional overhead.